libraries/AP_Mission/AP_Mission.h

```diff
@@ -454,6 +443,30 @@ class AP_Mission {
         uint8_t do_cmd_all_done : 1; // true if all "do"/"conditional" commands have been completed (stops unnecessary searching through eeprom for do commands)
     } _flags;
 
+    AP_Mission(AP_AHRS &ahrs, mission_cmd_fn_t cmd_start_fn, mission_cmd_fn_t cmd_verify_fn, mission_complete_fn_t mission_complete_fn) :
```

Comment:
To avoid more code churn. It's not true that it needs to be in the .cpp. If the construction is simple enough it can be left on the header and should be evaluated in a case by case manner.  Here it was simply because it was already there.

Reply:
When I move to .cpp it's either because then I can avoid include polutions or because it's not simple enough to be in the header. "Simple enough" being a very personal thing

Reply:
> When I move to .cpp it's either because then I can avoid include polutions

I consider that as needed to be in cpp 😉 

> because it's not simple enough to be in the header

Exactly the only ones I've commented on here. I'm not going to fuss here, but your *simple enough-o-meter* is terribly broken 😄